kaggle_cli/submit.py

```diff
@@ -46,7 +51,7 @@ def take_action(self, parsed_args):
         entry = parsed_args.entry
         message = parsed_args.message
 
-        archive_name = Submit._rand_str(10)+'.zip'
+        archive_name = Submit._rand_str(10) + '.zip'
```

Comment:
Since the uploaded file's name will also show on the Kaggle's dashboard. A related name might be better. i.e. append the uuid or timestamp to the original submission file's name.

And another concern there is, for myself habit, the submission files are already suffixed a timestamp to distinguish the trial. Adding additional suffix is duplicated.

Reply:
I agree. What I *prepend* a short random string instead?